index.js

```
@@ -66,24 +79,22 @@ function getStacks(app) {
-    return app._router.stack.reduce(combineStacks, []);
+    return app._router.stack.reduce(combineExpress4Stacks, []);
   }
 
   // Express 4 Router
   if (app.stack) {
-    return app.stack.reduce(combineStacks, []);
+    return app.stack.reduce(combineExpress4Stacks, []);
   }
 
   // Express 5
   if (app.router && app.router.stack) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)